actions/store_bot_client_info_MOD.js

```diff
@@ -135,6 +90,9 @@ module.exports = {
       case 46: // Last Message ID
         dataType = 'Last Message ID'
         break
+      case 47: // CPU Load Average
+        dataType = 'CPU Average Array'
```

Comment:
Use this or something similar
```suggestion
        dataType = 'Average CPU Usage Array'
```